app/modules/users/models.py

```diff
@@ -595,3 +597,12 @@ def delete(self):
         with db.session.begin():
             # TODO: Ensure proper cleanup
             db.session.delete(self)
+
+    @classmethod
+    def get_public_user(cls):
```

Comment:
this snuck in based on some stuff colin and i were talking about; really has no relevance until the PR related to owner/submitter which will happen shortly....